src/common/src/array/data_chunk_iter.rs

```diff
@@ -249,6 +249,19 @@ impl Row {
         &EMPTY_ROW
     }
 
+    /// Compare two rows' stream key
+    pub fn eq_by_pk(
+        row1: impl AsRef<Self>,
+        pk_indices1: &[usize],
+        row2: impl AsRef<Self>,
+        pk_indices2: &[usize],
+    ) -> bool {
```

Comment:
```suggestion
    pub fn eq_by_keys(
        row1: impl AsRef<Self>,
        key1: &[usize],
        row2: impl AsRef<Self>,
        key2: &[usize],
    ) -> bool {
```

Reply:
Actually, this function requires that the keys are ordered. In this case, would it be better named ordered_key?